include/boost/geometry/algorithms/area_result.hpp

```diff
@@ -73,15 +75,37 @@ struct default_area_result
         >
 {};
 
+// Workaround for VS2015
+#if (_MSC_VER < 1910)
```

Comment:
should it be explicitly `_MSC_VER == 1900` or there is hope that the workaround will also work with older than VS15 versions ?

Reply:
No, I don't think that the older VS will compile the library.

This way the checks are symmetric, i.e. `< 1910` vs `>= 1910`.